include/exadg/fluid_structure_interaction_precice/precice_parameter.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <deal.II/base/parameter_handler.h>
+
+using namespace dealii;
+
+/**
+ * This class declares all preCICE parameters, which can be specified in the
+ * parameter file. The subsection abut preCICE configurations is directly
+ * interlinked to the Adapter class.
+ */
+namespace Parameters
+{ /**
+   * @brief PreciceAdapterConfiguration: Specifies preCICE related information.
+   *        A lot of these information need to be consistent with the
+   *        precice-config.xml file.
+   */
+struct PreciceAdapterConfiguration
```

Comment:
I would write `PreciceParameters` (like the variable name `precice_parameters` in the drivers), without `Parameters` namespace